examples/quickstart.ipynb

```diff
@@ -0,0 +1,1146 @@
+     ]
+    }
+   ],
+   "source": [
+    "big_pandas_df.info()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Faster ``apply`` over a single column\n",
+    "\n",
+    "The performance benefits of Modin becomes aparent when we operate on large gigabyte-scale datasets. For example, let's say that we want to round up the number across a single column via the ``apply`` operation. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Time to apply with pandas: 43.969 seconds\n"
+     ]
+    }
+   ],
+   "source": [
+    "start = time.time()\n",
+    "rounded_trip_distance_pandas = big_pandas_df[\"trip_distance\"].apply(round)\n",
+    "\n",
+    "end = time.time()\n",
+    "pandas_duration = end - start\n",
+    "print(\"Time to apply with pandas: {} seconds\".format(round(pandas_duration, 3)))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Time to apply with Modin: 1.225 seconds\n"
+     ]
+    },
+    {
+     "data": {
+      "text/markdown": [
+       "### Modin is 35.88x faster than pandas at `apply` on one column!"
+      ],
+      "text/plain": [
+       "<IPython.core.display.Markdown object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "start = time.time()\n",
+    "\n",
+    "rounded_trip_distance_modin = big_modin_df[\"trip_distance\"].apply(round)\n",
+    "\n",
+    "end = time.time()\n",
+    "modin_duration = end - start\n",
+    "print(\"Time to apply with Modin: {} seconds\".format(round(modin_duration, 3)))\n",
+    "\n",
+    "printmd(\"### Modin is {}x faster than pandas at `apply` on one column!\".format(round(pandas_duration / modin_duration, 2)))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Summary\n",
+    "\n",
+    "Hopefully, this tutorial demonstrated how Modin delivers significant speedup on pandas operations without the need for any extra effort. Throughout example, we moved from working with 100MBs of data to 20GBs of data all without having to change anything or manually optimize our code to achieve the level of scalable performance that Modin provides.\n",
+    "\n",
+    "Note that in this quickstart example, we've only shown ``read_csv``, ``concat``, ``apply``, but these are not the only pandas operations that Modin optimizes for. In fact, Modin covers [more than 90\\% of the pandas API](https://github.com/modin-project/modin/blob/master/README.md#pandas-api-coverage), yielding considerable speedups for many common operations.\n",
+    "\n",
+    "Next, we discuss how Modin can be used locally on a single machine (e.g., your laptop!) and on a cluster setting."
```

Comment:
I think it is not clear `Next, we discuss how Modin ...`. When or where? Maybe it is worth removing this sentence?

Reply:
Fixed